jni/lib/src/jni_bindings_generated.dart

```diff
@@ -0,0 +1,3056 @@
+// Autogenerated file. Do not edit.
+// ignore_for_file: always_specify_types
+// ignore_for_file: camel_case_types
+// ignore_for_file: non_constant_identifier_names
+// ignore_for_file: constant_identifier_names
+// ignore_for_file: unused_field
+// ignore_for_file: unused_element
+
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+import 'dart:ffi' as ffi;
+
+/// Bindings for libdartjni.so which is part of jni plugin.
+///
+/// It also transitively includes type definitions such as JNIEnv from third_party/jni.h;
+///
+/// However, functions prefixed JNI_ are not usable because they are in a different shared library.
+///
+/// Regenerate bindings with `flutter pub run path_to_ffigen.dart --config ffigen.yaml`.
```

Comment:
`flutter pub run ffigen --config ffigen.yaml` and `dart run ffigen --config ffigen.yaml` should work without having to find a path to `ffigen.dart`. (It is the magic of `bin/<package-name>.dart`.)